publication/ver11/accessibilty.md

```diff
@@ -0,0 +1,274 @@
+textual information (titles) that can be used as a replacement.
+
+## If technology creates objects that don't have an inherent text representation
+### [yes] There is a mechanism to create short text alternatives that label the object.
+WoT TDs allow the embedding of textual titles and descriptions for interaction and Things.
+### [yes] There is a mechanism to create extended text alternatives for fallback content.
+Both short and long text (title and descriptions) are supported.
+In addition, links to external documentation can be referenced for the entire Thing.
+### [no] Text alternatives can be semantically "rich" e.g., with page structure, text style, hyperlinks, etc.
+Unfortunately, HTML markup is forbidden in embedded text to avoid an XSS security risk.
+Titles and descriptions, for example, might be used to generate an HTML dashboard but embedding text from
+a Thing Description into the HTML for such a dashboard would allow an attacker to also
+embed undesirable scripts and other malicious content.
+If rich formatting is desired it can, however, be included in linked documentation.
+
+## If technology provides content fallback mechanisms, whether text or other formats
+### [no] Authors can explicitly mark content as not needing alternative content because it does not perform an important role.	
+### [no] Content can explicitly indicate when author declined to provide alternative content.	
+### [no] Content can explicitly indicate that authoring tool is unable to generate or obtain alternative content.	
+### [no] Authors can explicitly associate alternative content with the primary content.	
+### [yes] Authors can associate multiple types and instances of alternative content with primary content.
+All titles and descriptions can be given in multiple languages.
+### [n/a] Alternate content can be easily found from the initial content.	
+No alternative content, per se.
+
+## If technology provides visual graphics
+This is a developing area, being explored by the SVG Accessibility Task Force.	
+
+## If technology provides internationalization support
+### [yes] Accessibility features can be internationalized to the same degree as other features.
+There are, however, no features exclusively targetting accessiblity.
+
+## If technology defines accessible alternative features
+### [n/a] Accessible alternatives themselves meet the same bar of accessibility.
+No special features exclusively targetting accessiblity are provided.
+
+## If technology provides content directly for end-users
+### [yes] Content can be encoded in a manner that allows machine transformation into accessible output.
+Content is generally either text or JSON.  Text can be rendered into speech if necessary,
+and JSON provides structured data that can be transformed into an accessible output.
+
+## If technology defines an API
+A Scripting API is defined for WoT in an associated informative document but its use is not
```

Comment:
we can add the link to the scripting API https://www.w3.org/TR/wot-scripting-api/